dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityAttribute.java

```diff
@@ -424,10 +426,22 @@ public void setFormName( String formName )
     {
         this.formName = formName;
     }
-    
+
     public Boolean isSystemWideUnique()
     {
         return isUnique() && !getProgramScope() && !getOrgunitScope();
     }
 
+    @JsonProperty
+    @JacksonXmlProperty( namespace = DxfNamespaces.DXF_2_0 )
+    public Boolean getSkipSynchronization()
+    {
+        return skipSynchronization != null ? skipSynchronization : false;

```

Comment:
You should not do a null-check here.

false should be default(?) and during startup routines, any null values should be set to the default value. In other words at this point it should be either true or false, never null.

Reply:
Fixed